samtranslator/model/api/api_generator.py

```diff
@@ -208,6 +213,73 @@ def _add_cors(self):
         # Assign the Swagger back to template
         self.definition_body = editor.swagger
 
+    def _add_auth(self):
```

Comment:
Note that additional auth settings such as ResourcePolicy and IAM will be set here also when they're added.

Reply:
Can you either leave a note here or in the corresponding issue making note of this?